students/medrek_tomasz/lesson_03_functions/fibonacci_numbers.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+
+def fibonacci(n):
+    if (n == 0) or (n == 1):
+        return n
+    else:
+        return fibonacci(n - 1) + fibonacci(n - 2)
+
+
+try:
```

Comment:
You can put it inside while loop

Reply:
https://snakify.org/lessons/functions/problems/fibonacci_rec/
"Don't use loops, use the flair of recursion instead. However, you should think about why the recursive method is much slower than using loops. "